src/cloud/project/services.md

```diff
@@ -163,12 +163,15 @@ To verify relationships in remote environments:
 
 ## Service versions
 
-The following table lists the services used in {{site.data.var.ece}} and their version compatibility with the [Magento Cloud template](https://github.com/magento/magento-cloud).
+The following table lists the services used in {{site.data.var.ece}} and their version compatibility with the [Magento Cloud template](https://github.com/magento/magento-cloud). 
+
+{:.bs-callout-info}
+Version support and compatibility for Magento Commerce on Cloud is determined by service versions deployed on the Cloud infrastructure. In some cases, the supported or recommended versions differ from the versions supported by Magento Commerce depoloyed on-premises.
```

Comment:
I like this note and I think it could be useful elsewhere, like dropped in a core topic about Elasticsearch 😉  . It might be a candidate for include.

I only have one concern about "on-premises". Usually, the dash makes a phrase like that an adjective. I think here, perhaps the dash is not necessary? Or could reword "the Magento Commerce on-premises deployment." Not sure.